subiquity/common/types.py

```diff
@@ -386,6 +386,12 @@ class StorageResponseV2:
     install_minimum_size: Optional[int] = None
 
 
+class GuidedCapability(enum.Enum):
+    DIRECT = enum.auto()
```

Comment:
I find a string value easier to debug on the JSON dumps, if you don't mind using that in place of the integers `auto()` is going to generate.

Reply:
we serialize enums by name not value (on the subiquity api at least)